### PR TITLE
Migrate Morpher to Chakra

### PR DESCRIPTION
**Pull request by laurentlucian**
## Description

- Migrates `Morpher` component from `emotion` to `chakra-ui`

## Related Issue

#6374

### CODE REVIEW

File: src/components/Morpher.tsx [hunk 1]
```diff
@@ -1,20 +1,6 @@
...
 
-const NavLink = styled(Link)`
-  text-decoration: none;
-  font-size: 1rem;
-  color: ${(props) => props.theme.colors.text};
-  &:hover {
-    text-decoration: none;
-    color: ${(props) => props.theme.colors.primary};
-  }
-  &.active {
```
Comment (laurentlucian) [inline src/components/Morpher.tsx:13]: I couldn't tell what the `.active` class is doing so I couldn't compare, but decided to leave as is as it's not in the scope of this PR to fix this

Comment (pettinarip) [reply src/components/Morpher.tsx:13]: I think that is fine. We were not using it before. This class is related to the active link class that Gatsby allows you to pass. https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#link-api-surface-area

File: src/components/Morpher.tsx [hunk 2]
```diff
@@ -154,9 +140,18 @@ const Morpher = () => {
...
 
   return (
-    <NavLink to="/languages/">
+    <Link
+      textDecor="none"
+      fontSize="1rem"
+      color="text"
+      _hover={{ color: "primary" }}
+      sx={{
+        "&.active": { fontWeight: "bold" },
```
Comment (pettinarip) [inline src/components/Morpher.tsx:149]: Lets remove this for now since it is not doing anything

**Review by pettinarip — CHANGES_REQUESTED** [review]
Cool, thanks for the PR @laurentlucian 💪🏼

File: src/components/Morpher.tsx [hunk 3]
```diff
@@ -154,9 +140,18 @@ const Morpher = () => {
   }, [])
 
   return (
-    <NavLink to="/languages/">
+    <Link
+      textDecor="none"
+      fontSize="1rem"
```
Comment (pettinarip) [inline src/components/Morpher.tsx:145]: could we do
```suggestion
      fontSize="md"
```

**Review by pettinarip — CHANGES_REQUESTED** [review]

**Review by pettinarip — APPROVED** [review]
🚀 great! ty @laurentlucian!

**Pull request merged**